qase-playwright/changelog.md

```diff
@@ -1,8 +1,35 @@
+# playwright-qase-reporter@2.0.0-beta.6
+
+## What's new
+
+Add functionality for capture stdout and stderr logs as attachments. 
+You can enable this feature by setting the `captureLogs` option to `true` in the reporter configuration:
```

Comment:
```suggestion
Capture `stdout` and `stderr` logs as attachments. 
To enable this feature, add `captureLogs: true` to the reporter configuration:
```

Reply:
yes, we can use ENV and file configs. JS reporters don't have CLI parameters